src/RouteList.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const Ioc = require('adonis-fold').Ioc
+const Command = Ioc.use('Adonis/Src/Command')
```

Comment:
I've changed and pushed now.

And I've made an improvement adding Action:

![adonisjs-command-r-list-39423482](https://cloud.githubusercontent.com/assets/304086/18315888/f7ec9d96-74ee-11e6-9ad1-6b4c8ebc8744.png)